src/app/swap/swap-out.ts

```diff
@@ -0,0 +1,24 @@
+import { BTC_NETWORK } from "@config"
+import { TxDecoder } from "@domain/bitcoin/onchain"
+import { SwapServiceError } from "@domain/swap/errors"
+import { SwapOutResult } from "@domain/swap/index.types"
+import { SwapService } from "@services/swap"
+import { OnChainService } from "@services/lnd/onchain-service"
+
+export const swapOut = async ({
+  amount,
```

Comment:
when there is just argument we typically don't use an object, but just pass the scalar directly (ie :`(amount)` instead of `({amount})`)